sympy/functions/special/tests/test_delta_functions.py

```diff
@@ -139,3 +139,12 @@ def test_rewrite():
     assert 5*x*y*Heaviside(y + 1).rewrite(SingularityFunction) == 5*x*y*SingularityFunction(y, -1, 0)
     assert ((x - 3)**3*Heaviside(x - 3)).rewrite(SingularityFunction) == (x - 3)**3*SingularityFunction(x, 3, 0)
     assert Heaviside(0).rewrite(SingularityFunction) == SingularityFunction(0, 0, 0)
+
+def test_issue_15923():
```

Comment:
I have added the `test_issue_15293()` to the `test_delta_functions.py` but shouldn't I add the assertion code for regression test in the `test_rewrite()` method in the same file `test_delta_functions.py`